probe-rs/src/probe/jlink/mod.rs

```diff
@@ -909,9 +909,8 @@ impl DebugProbe for JLink {
     }
 
     fn target_reset(&mut self) -> Result<(), DebugProbeError> {
-        Err(DebugProbeError::NotImplemented {
-            function_name: "target_reset",
-        })
+        self.write_cmd(&[Command::ResetTarget as u8])?;
```

Comment:
Afaik this is meant to be hardware reset.

Reply:
Maybe, I just trusted what [I found](https://github.com/probe-rs/probe-rs/issues/993#issuecomment-1028273732) :)

Reply:
Okay so I'm not sure what the issue is. `reset_target` resets the target device 🙃 CMSIS-DAP asserts the nRESET line for some time (though the doc comment doesn't mention deassert, I think it's implied), ST-Link does the same, W-Link sends `ResetTarget` and now so does J-Link.

Reply:
@Yatekii is there anything you'd like me to do, or can you tell me what exactly your comment referred to?